src/ApparentHorizons/Tags.hpp

```diff
@@ -325,6 +326,23 @@ struct SurfaceIntegral : db::ComputeTag {
   using argument_tags = tmpl::list<AreaElement<Frame>, IntegrandTag,
                                    StrahlkorperTags::Strahlkorper<Frame>>;
 };
+/// Tag representing the surface area of a Strahlkorper
```

Comment:
Given the length of this file, is there another new file that could house the horizon quantity tags? I find files with this many different things in them a bit difficult to follow. @geoffrey4444 do you have any naming ideas or thoughts on this?

Reply:
StrahkorperGr_Tags.hpp?

Reply:
On second thoughts, from a user’s point of view, it might be annoying to figure out if an AH tag is a GR tag or not...

Maybe the compute tags should move to to StrahlkorperGr.hpp, and the simple tags stay in Tags.hpp?

Reply:
Actually, I think `StrahlkorperGr` should be split into files, one for each function. I think then it's reasonable to put the compute tags with the functions. That file is also quite long and the name is not as descriptive as they would be if the functions were in different files.

Reply:
OK, but let's handle this factoring in another PR, not one we give my students? Assign me an issue?

Reply:
Could you make the issue and assign yourself?

Reply:
#2334